torch/quantization/quantize.py

```diff
@@ -429,7 +451,10 @@ def _convert(module, mapping=None, inplace=False):
                          nniqat.ConvBnReLU2d)
 
     for name, mod in module.named_children():
-        if type(mod) not in SWAPPABLE_MODULES:
+        # both swappable modules and observed custom modules are
+        # swapped as one unit
+        if type(mod) not in SWAPPABLE_MODULES and \
```

Comment:
When we support custom module fusion, we will need to change code here. Currently, it looks like Swappable modules defines the list of fusions that are supported. Thoughts on how this can be addressed? There main use case would be : nn.module A + nn.module B => Custom Fused Module C

Reply:
we can make this configuration with registrations as well, can be done in a separate PR before we need this feature